adagrams/game.py

```diff
@@ -1,11 +1,82 @@
+import random
+import string
+
+value_of_letters = {
+        
+        "A" : 1 , "B" : 3 , "C": 3 , 
+        "D" : 2 , "E": 1 , "F": 4 ,
+        "G" : 2 , "H" : 4, "I" : 1, 
+        "J" : 8 , "K" : 5 ,"L" : 1 , 
+        "M" : 3 , "N" : 1 , "O" : 1 , 
+        "P": 3, "Q" : 10 , "R" : 1 ,
+        "S" : 1 , "T" : 1 , "Q" : 10 ,
+        "U" : 1 , "V" : 4 , "W" : 4 , 
+        "X" : 8 , "Y" : 4 , "Z" : 10
+        
+        }
+    
+
 def draw_letters():
-    pass
+    
+    pool_of_letters = {
+    
+    
```

Comment:
Delete unnecessary white lines here to keep your project files concise. Also white lines on lines 5, 15, 28, 30-31 should be deleted too.

Here's the PEP8 guidance on blank lines: https://peps.python.org/pep-0008/#blank-lines